wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -123,10 +123,10 @@ <h3>{% if postelection.is_london_assembly_additional %}Additional members{% else
                                     winner{{ postelection.people|pluralize }}.
                                     A new election to fill the unclaimed seat{{ postelection.people|pluralize }}
                                     will be held within 35 working days of the original election date.</p>
-                            {% else %}
+                            {% elif postelection.people.count == 0 %}
```

Comment:
I added this to be explicit, but recognise that an `elif` without an `else` doesn't work. Will refactor with fresh eyes. 👀